### PR TITLE
[Cisco ISE] Add event.code based on cisco_ise.log.message.code

### PR DESCRIPTION
**Pull request by philippkahr**
Added the ECS field `event.code` based on feedback from our security consultants. @georgessamaha 

I opted for a `set` instead of a `rename` for the field `cisco_ise.log.message.code` to ensure that we don't break any existing functionality from customers.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/cisco_ise/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 1.18.0
+  changes:
+    - description: Add field event.code to capture the cisco_ise.log.message.code.
+      type: enhancement
+      url: https://github.com/elastic/integrations/pull/8071
```
Comment (efd6) [inline packages/cisco_ise/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/8071
```

**Review by bhapas — COMMENTED** [review]
Looks like the public documentation does not show up `event.code` . Can you rebuild integration locally so that docs get regenerated.
https://docs.elastic.co/integrations/cisco_ise

**Comment by philippkahr** [comment]
@bhapas should be fixed now :)

**Review by bhapas — APPROVED** [review]
LGTM

**Pull request merged**